### PR TITLE
[grafana] Added required parameter to example documentation

### PR DESCRIPTION
**Pull request by TheDruidsKeeper**
The image-render requires the [default](https://github.com/grafana/grafana-image-renderer/blob/v3.0.1/src/config.ts#L65) `--no-sandbox,` parameter to run in headless mode, or it [fails](https://bugs.chromium.org/p/chromium/issues/detail?id=638180).

### CODE REVIEW

**Comment by zanhsieh** [comment]
@TheDruidsKeeper 
Could you sign DCO and update the chart version to latest please?

**Comment by TheDruidsKeeper** [comment]
@zanhsieh Done.

**Review by zanhsieh — APPROVED** [review]

File: charts/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.16.2
+version: 6.16.3
```
Comment (zanhsieh) [inline charts/grafana/Chart.yaml:3]: Now should be 6.16.4.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/grafana/Chart.yaml (hunk 1 above)
Comment (TheDruidsKeeper) [reply charts/grafana/Chart.yaml]: Updated to 6.16.6 (currently the next version)

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**